docs/sboms/oci_image.md

```diff
@@ -47,19 +47,146 @@ mobster --verbose  generate oci-image \
 - `--dockerfile-target` -- if a build target was used for multi-stage build, use this argument to specify the build target
 - `--additional-base-images` -- optionally add references to other build images outside the parsed Dockerfile.
   expects the format `<registry>/<repository>:<tag>@sha256:<digest value>`
-- `--contextualize` -- NOT IMPLEMENTED YET
+- `--contextualize` -- Allows SBOM contextualization (see [Contextual SBOM](#contextual-sbom))
 - `--output` -- where to save the SBOM. prints it to STDOUT if this is not specified
 
 
-# Structure of the generated SBOM
+## Contextual SBOM
+
+Contextual SBOM is a feature that establishes relationships between container images and their parent
+(base) or builder images in the software supply chain. Instead of treating each
+image as isolated, it creates a hierarchical view showing how packages flow from
+parent or builder images to child images.
+
+
+* Feature will be released gradually. Planned features:
+    * ✅ **Contextual SBOM for non-hermetic build**
+        * identification of the base image content in component in non-hermetic build
+        * achieved by matching packages from the base image SBOM to the component
+          SBOM and marking their origin with relationships
+    * ⏳ **Builder content contextualization**
+        * identification of the content copied from multistage build stages to component
+        * acquired by generating an SBOM from content copied from builder images
+          and merged into the final SBOM to indicate the origin of that content
+    * ⏳ **Contextual SBOM for hermetic build**
+        * differentiation of the base image content and component content in hermetic build
+        * assembled from base image content, hermeto content and - in multistage builds -
+          builder content
+
+
+## Requirements for executing Contextual SBOM workflow
+* Base image of the component must be built by konflux and must have SBOM attached
+* Attached base image SBOM must be in SPDX format
+* --contextualize flag is present
+
+If requirements are not fulfilled, legacy SBOM (not contextualized) will be produced.
+
+
+## Functional breakdown
+* Feature profoundly changes relationship structure of the SBOM
+    * **Legacy SBOM:** Relationships capture only first base image as component ancestor.
```

Comment:
You use the term `legacy` across multiple places in this document. For some people, this might sound like something that is bad, and they should avoid it. Since contextual SBOM is another feature that changes the format of the SBOM, I would avoid calling the previous version legacy. Could you rephrase it to something more specific - for example, non-contextual SBOM or any similar variant?